transaction/transfer_test.go

```diff
@@ -14,12 +14,14 @@ func TestTransferProcess_SufficientFunds(t *testing.T) {
 		Begin(t).
 		Prepare(
 			commands.OpenAccount{
-				AccountID: "A001",
-				Name:      "Anna",
+				CustomerID:  "C001",
+				AccountID:   "A001",
+				AccountName: "Savings",
```

Comment:
Not sure if this is worth mentioning, but the "account name" is intended to be the kind of account name you see on a real bank account, usually the customer's name, or a business name, that kind of thing.

Reply:
Ok yeah. I wasn't quite sure to even keep those after adding the customer that has a name. It makes sense to keep them and use an actual name instead of a kind of account.